databases/week2/database_lesson2.txt

```diff
@@ -0,0 +1,31 @@
+-- 1. Get all the tasks assigned to users whose email ends in @spotify.com
+-- with using aliases:-
+SELECT *
+FROM task t
+INNER JOIN user_task ut ON t.id = ut.task_id
+INNER JOIN user u ON ut.user_id = u.id
+WHERE u.email LIKE '%@spotify.com';
+
+-- 2. Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT t.title
+	,s.NAME
+FROM task t
+INNER JOIN user_task ut ON t.id = ut.task_id
+INNER JOIN user u ON ut.user_id = u.id
+INNER JOIN STATUS s ON t.status_id = s.id
+WHERE u.NAME = 'Donald Duck'
+	AND s.NAME = 'Not started';
```

Comment:
Case sensitivity `STATUS` is not like `status` when working from VScode. Please fix it to align with the rest of your code.

Reply:
I will ,Thank you Elias